static/js/toolbar.js

```diff
@@ -90,7 +90,7 @@ exports.postToolbarInit = (hook, context) => {
         $('#imageUploadModalLoader').show();
         $.ajax({
           type: 'POST',
-          url: `/p/${clientVars.padId}/pluginfw/ep_image_upload/upload`,
+          url: `${clientVars.urlPrefix}${clientVars.padId}/pluginfw/ep_image_upload/upload`,
```

Comment:
Wouldn't it be better to use a relative URL? Like this:

```suggestion
          url: 'pluginfw/ep_image_upload/upload',
```

Reply:
Yes, I think it's a good idea to be able to configure a relative URL. But I guess you want to keep the `clientVars.padId` so the server can organize the images into the respective image folders. So if you want to have a relative URL you can just set `urlPrefix: '',` in the settings.json. This will lead to e.g. `g.s8oes9dhwrvt0zif$test/pluginfw/ep_image_upload/upload` for the url

Reply:
Whoops you're right—I forgot that pad URLs don't have a trailing slash. This should take care of it without the need for a new client var:

```suggestion
          url: `${clientVars.padId}/pluginfw/ep_image_upload/upload`,
```

Reply:
Yeah, we could remove the clientvar. I also removed the lines in the readme